starters/http/dotnet/infra/app/processor.bicep

```diff
@@ -25,6 +27,7 @@ module processor '../core/host/functions-flexconsumption.bicep' = {
     appSettings: union(appSettings,
       {
         AzureWebJobsStorage__clientId : identityClientId
+        APPLICATIONINSIGHTS_AUTHENTICATION_STRING: applicationInsightsIdentity
```

Comment:
thanks i like this better here in /app versus in /core

Reply:
:-)